ui/guest/src/store/reducers/root.ts

```diff
@@ -130,12 +130,13 @@ const reducer = createReducer(initialState, {
       } else if (highlightMode === HighlightMode.MOVE_TARGETS) {
         const iceId = record.iceIds[0];
         const movableRecordId = iceRegistry.getMovableParentRecord(iceId);
-        if (notNullOrUndefined(movableRecordId)) {
+        const movableRecord = fromICEId(movableRecordId);
+        if (notNullOrUndefined(movableRecord)) {
           const draggable = getDraggable(movableRecordId);
           const highlight = getHoverData(
             // If (iceId == movableRecordId) the current record is already
             // the one to show the highlight on.
-            iceId === movableRecordId ? record.id : fromICEId(movableRecordId).id
+            iceId === movableRecordId ? record.id : movableRecord.id
```

Comment:
Please explain these changes. What's wrong with checking movableRecordId without pulling the element record yet?

Reply:
we can revert this change, i was trying to fix the issue when the items are moved and the registration is wrong